config.yaml

```diff
@@ -52,24 +52,26 @@ commands:
     rules: [ 'allow' ]
 templates:
   comps_list:
-    slack: |
-      {{name}}
+    body: |
+      ~$results[0].name~
   comps_status:
-    slack: |
-      Component: {{name}}  Status: {{status}}  Last Updated: {{last_updated}}
+    body: |
+      Component: ~$results[0].name~  Status: ~$results[0].status~  Last Updated: ~$results[0].last_updated~
```

Comment:
This is fine, but it'd be a little nicer to wrap in an each in the case that you run `comps_list | comps_status`